test/karma.conf.js

```diff
@@ -144,6 +151,11 @@ module.exports = function setKarmaConfig(config) {
         },
       },
     };
+
+    // default 1000, Avoid Rate Limit Exceeded
+    newConfig.pollingTimeout =
+      ((MAX_KARMA_CONCURRENT_BUILD * (newConfig.browsers.length - 1)) / MAX_REQUEST_BROWSERSTACK) *
```

Comment:
`newConfig.browsers.length - 1` should be "BROWSERSTACK_BROWSERS_USED". Otherwise this looks like an off-by-one error.